src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../types/comment';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <>
+    {comments.length ? (
+      <div className="CommentList">
+        {comments
+          .map((comment) => <CommentInfo key={comment.id} comment={comment} />)}
+      </div>
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    )}
```

Comment:
let's move this check to PostInfo component where you render CommentList component